web/index.html

```diff
@@ -7,6 +7,9 @@
   </head>
   <body>
     <div id="root"></div>
+    <!-- <div class="loader" id="app">
```

Comment:
Uhh, there's never a reason to edit this file this way

Reply:
now be me o

Reply:
I had that feeling :laughing: